app/models/webhook/endpoint.rb

```diff
@@ -1,13 +1,19 @@
 module Webhook
   class Endpoint < ApplicationRecord
     belongs_to :user, inverse_of: :webhook_endpoints
+    # rubocop:disable Rails/InverseOf
+    belongs_to :oauth_application, optional: true,
+                                   class_name: "Doorkeeper::Application",
+                                   foreign_key: :oauth_application_id
+    # rubocop:enable Rails/InverseOf
```

Comment:
I'm assuming this rubocop lint is from missing a `has_many/has_one` on OauthApplication's side. I'm not certain if this is the right way to go about it, but I had to [add additional relation to doorkeeper.rb](https://github.com/thepracticaldev/dev.to/pull/3504#discussion_r305168262) to get it working on my initial Doorkeeper implementation.

Reply:
Yeah, we should add the inverse relation if we can